Lib/xml/etree/ElementTree.py

```diff
@@ -120,8 +120,9 @@ class ParseError(SyntaxError):
 
 def iselement(element):
     """Return True if *element* appears to be an Element."""
-    return hasattr(element, 'tag')
-
+    return (hasattr(element, 'tag') and hasattr(element, 'text') and
+        hasattr(element, 'tail') and callable(element.iter) and
+        callable(element.items) and callable(element.__len__))
```

Comment:
Does this function raise an exception if iter(), items() or `__len__()` method doesn't exist?